### DISCUSSION TITLE
PVC size in longhorn UI is different than actual size used by Pod.

### DISCUSSION THREAD

**Discussion by BankarAkanksha17** [longhorn/longhorn#8779] [General]
Hi Team,

PVC size in longhorn is way more larger than actual size of that particular PVC in pod why this is happening?

Please Help.

<img width="948" alt="image" src="https://github.com/longhorn/longhorn/assets/170407001/6adb3239-ca21-490e-8aa0-bf0d9eb705e5">

this is size of the PVC taken in longhorn.
<img width="400" alt="image" src="https://github.com/longhorn/longhorn/assets/170407001/264f2318-a957-489b-ad40-2a6a2808812b">
this is the size of that mounted directory inside the pod.
<img width="727" alt="image" src="https://github.com/longhorn/longhorn/assets/170407001/66fbbb3c-f509-4990-9e24-817b115f387f">
<img width="646" alt="image" src="https://github.com/longhorn/longhorn/assets/170407001/a9be9b29-c06c-4e6e-8545-43ecb018295e">
and this is the large file store in longhorn diskspace pls explain what exactly store in this file which is so large and how to cleanup unwanted data in this file.

Thanks.

**Comment by derekbit** [comment]
Please see the doc https://longhorn.io/docs/1.6.2/nodes-and-volumes/volumes/volume-size/.
